tests/rptest/services/admin_ops_fuzzer.py

```diff
@@ -497,8 +499,8 @@ def execute_with_retries(self, op_type, op):
                 return op.execute(self.operation_ctx)
             except Exception as e:
                 error = e
-                self.redpanda.logger.info(
-                    f"Operation: {op_type} error: {error}, retries left: {self.retries-retry}/{self.retries}"
+                self.redpanda.logger.exception(
+                    f"Operation: {op_type}, retries left: {self.retries-retry}/{self.retries}"
```

Comment:
similar here (to my reply on the comment https://github.com/redpanda-data/redpanda/pull/7903/files#r1055733582). presumably INFO level is used because it's being handled--an exception occurred, but it isn't an error. So instead of logging at ERROR level (which is what `.exception` does), the exception context can just be added to existing INFO level messages with an extra keyword argument (i think `exc_info=True`). cc @mmaslankaprv